source/type/data.cpp

```diff
@@ -426,9 +426,23 @@ void Data::build_associated_calendar() {
     }
 }
 
+static void rel_stop_point_network(nt::VehicleJourney* vj){
+    // network of stop_point
+    for (nt::StopTime& stop_time: vj->stop_time_list){
```

Comment:
can't it create problems ?

can't a stoppoint be a part of several networks ? thus do we need a network field in a stoppoint ?